dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task230110MakeSomeSystemFieldsRemovableByVarNameTest.java

```diff
@@ -0,0 +1,76 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.common.db.DotConnect;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Map;
+
+import static com.dotmarketing.db.DbConnectionFactory.isDBFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Verifies that the {@link Task230110MakeSomeSystemFieldsRemovableByVarName} UT runs as expected.
+ *
+ * @author Jose Castro
+ * @since Jan 10th, 2023
+ */
+public class Task230110MakeSomeSystemFieldsRemovableByVarNameTest {
+
+    private static final String PAGE_ASSET_BASE_TYPE_ID = "c541abb1-69b3-4bc5-8430-5e09e5239cc8";
+    private static final String FILE_ASSET_BASE_TYPE_ID = "33888b6f-7a8e-4069-b1b6-5c1aa9d0a48d";
+    private static final String PERSONA_BASE_TYPE_ID = "c938b15f-bcb6-49ef-8651-14d455a97045";
+    private static final String HOST_TYPE_ID = "855a2d72-f2f3-4169-8b04-ac5157c4380c";
+
+    private static final Map<String, List<String>> CONTENT_TYPES_AND_FIELDS = Map.of(
```

Comment:
This needs to be BASE_TYPES and Fields.  We need to change these values in all the content types of a given BASE_TYPE.  For example, in our starter, we have 4 different "HTMLPAGE" types.  The fields need to be changed in each one of these types:




![Screen Shot 2023-01-12 at 12 17 58 PM](https://user-images.githubusercontent.com/934364/212135316-a5325a3e-f81a-4638-9efd-29a291b3b54e.png)

Reply:
Noted.

Reply:
Done!